File: FCSeafood/FCSeafood.BusinessObjects/Models/Auxiliary/BindCategoryModel.cs
using FCSeafood.BusinessObjects.Models.Common;

namespace FCSeafood.BusinessObjects.Models.Auxiliary;

public class BindCategoryModel {
    public CategoryTModel CategoryTModel { get; set; } = new();
    public SubcategoryTModel SubcategoryTModel { get; set; } = new();
}

File: Angular/FCSeafood/src/app/common/data/common/models/common/BindCategoryModel.ts
import {CategoryTModel} from "@common-data/common/models/common/category-type.model";
import {SubcategoryTModel} from "@common-data/common/models/common/subcategory-type.model";

export class BindCategoryModel {
  categoryTModel: CategoryTModel = new CategoryTModel();
  subcategoryTModel: SubcategoryTModel = new SubcategoryTModel();
}
